docs/local.rst

```diff
@@ -43,7 +43,7 @@ Configuration
 
     tutor config save --interactive
 
-This is the only non-automatic step in the installation process. You will be asked various questions about your Open edX platform and appropriate configuration files will be generated. If you would like to automate this step then you should run ``tutor config save --interactive`` once. After that, there will be a ``config.yml`` file at the root of the project folder: this file contains all the configuration values for your platform, such as randomly generated passwords, domain names, etc.
+This is the only non-automatic step in the installation process. You will be asked various questions about your Open edX platform and appropriate configuration files will be generated. If you would like to automate this step then you should run ``tutor config save --interactive`` once. This will generate a ``config.yml`` file in the **project root**. The location of the **project root** can be found by running ``tutor config printroot``. See (section above)[#tutor-root].
```

Comment:
Can you preserve this information?

> this file contains all the configuration values for your platform, such as randomly generated passwords, domain names, etc.

This link will not render correctly: `See (section above)[#tutor-root]`. You should use a `:ref:` -- see other places in the docs for reference. Please build the docs to check that your changes render correctly.